torch/csrc/jit/passes/quantization.cpp

```diff
@@ -32,14 +32,15 @@ void fillQConfigMap(
   }
   map[module.module_object()] = qconfig;
 
-  for (script::Slot s : module.get_module_slots()) {
+  for (auto p : module.get_modules()) {
```

Comment:
This code is less understandable than before. It replaced a named type with auto, and replaced informative function names with first and second.

Reply:
> This code is less understandable than before. It replaced a named type with auto, and replaced informative function names with first and second.

Sure, I'll replace pairs with structs.

But that said, I am not completely sold on "code with pairs is less readable than code with a custom arbitrary type". Take into account that you are already familiar with the code, and you know what `Slot` is - that's why that version is more readable to you. However, for new people (e.g. for me) code with `.first` and `.second` is much easier to understand because I immediately know what it means in contrast to a code that uses `Slot` with 10 methods to operate on its state.

Having a simple structure with no methods seems to be a good choice in the middle though.